Version 0.5.18 fails on OSX Yosemite

A black window appears with a working file menu.

```
$ /Applications/Kitematic\ \(Beta\).app/Contents/MacOS/Kitematic\ \(Beta\) 
[60309:0421/174251:INFO:renderer_main.cc(212)] Renderer process started
[60307:0421/174252:INFO:CONSOLE(109)] "Download the React DevTools for a better development experience: http://fb.me/react-devtools", source: /Applications/Kitematic (Beta).app/Contents/Resources/app/node_modules/react/lib/React.js (109)
[60307:0421/174252:INFO:CONSOLE(239)] "[Bugsnag] Invalid API key 'undefined'", source: /Applications/Kitematic (Beta).app/Contents/Resources/app/node_modules/bugsnag-js/src/bugsnag.js (239)
[60307:0421/174252:INFO:CONSOLE(326)] "Uncaught Error: Cannot find module 'classNames'", source: module.js (326)
$ uname -a
Darwin host.local 14.1.0 Darwin Kernel Version 14.1.0: Mon Dec 22 23:10:38 PST 2014; root:xnu-2782.10.72~2/RELEASE_X86_64 x86_64
```

--- a/src/ContainerDetailsSubheader.react.js
+++ b/src/ContainerDetailsSubheader.react.js
@@ -9,7 +9,7 @@ var ContainerUtil = require('./ContainerUtil');
 var machine = require('./DockerMachine');
 var RetinaImage = require('react-retina-image');
 var webPorts = require('./Util').webPorts;
-var classNames = require('classNames');
+var classNames = require('classnames');
 
 var ContainerDetailsSubheader = React.createClass({
   contextTypes: {

